src/Form/Admin.php

```diff
@@ -78,19 +78,29 @@ public function getFormId() {
   public function buildForm(array $form, FormStateInterface $form_state) {
     $config = $this->config('islandora_spreadsheet_ingest.settings');
     $current_whitelist = $config->get('binary_directory_whitelist');
+    $scheme_options = $this->streamWrapperManager->getNames(StreamWrapperInterface::READ_VISIBLE) + [
+      'http' => 'HTTP',
+      'https' => 'HTTPS',
+      'ftp' => 'FTP',
+    ];
     $form['schemes'] = [
       '#type' => 'checkboxes',
       '#title' => $this->t('Schemes'),
       '#description' => $this->t('Allowed list of schemes for which binaries can be referenced from.'),
       '#default_value' => $config->get('schemes') ?? [],
-      '#options' => $this->streamWrapperManager->getNames(StreamWrapperInterface::READ_VISIBLE),
+      '#options' => $scheme_options,
     ];
     $form['paths'] = [
       '#type' => 'textarea',
       '#title' => $this->t('Binary path whitelist'),
       '#default_value' => $current_whitelist ? implode(',', $current_whitelist) : '',
       '#description' => $this->t('A comma separated list of local locations from which spreadsheet ingests can use binaries.'),
     ];
+    $form['info'] = [
+      '#type' => 'item',
+      '#title' => $this->t('If nothing is whitelisted, everything is whitelisted.'),
+      '#description' => $this->t('The plugin that expects to use this information will log warnings if this configuration is missing.'),
+    ];
```

Comment:
Just had a thought, but: Do we want to actually document this as behaviour? If we just consider this pass-through behaviour as a deprecation allowance, with the intent to stop allowing it in a new major version or something, then it might not make sense to document?